### PR TITLE
Uncaught ReferenceError Fix: wrap Drift account ID in quotes

### PR DESCRIPTION
**Pull request by sassela**
Fixes issue with https://github.com/railslove/rack-tracker/pull/139

Thank you in advance!

### CODE REVIEW

**Comment by bumi** [comment]
damn, guess I should have done a better review. 
thanks for the fix. can you maybe adjust a test for that?

**Comment by sassela** [comment]
Sure, is this sufficient? Screenshot shows the test case failing on master and passing on this branch with the change. 

Happy to extend further if needed

Screenshot 2019-07-31 at 17 09 31

File: spec/integration/drift_integration_spec.rb [hunk 1]
```diff
@@ -13,6 +13,6 @@
 
   it 'embeds the script with account_id' do
     expect(page.find('script')).to have_content('js.driftt.com')
-    expect(page.find('script')).to have_content('DRIFT_ID')
+    expect(page.find('script')).to have_content('\'DRIFT_ID\'')
```
Comment (bumi) [inline spec/integration/drift_integration_spec.rb]: maybe we check for the whole `drift.load('DRIFT_ID');` to be sure we actually check for that and not another DRIFT_ID?

Comment (sassela) [reply spec/integration/drift_integration_spec.rb]: Was in a meeting, but have updated the check now

**Comment by DonSchado** [comment]
will prepare the release this evening

**Comment by DonSchado** [comment]
@sassela please check the new release 1.11.1 :)

**Comment by bumi** [comment]
thanks @DonSchado 

1.11.1 is the best release! :shipit:

**Comment by sassela** [comment]
Thanks to you both! Much appreciated

**Pull request merged**